### ISSUE TITLE
[Feature Request] Go back to previous view

### ISSUE THREAD

**Issue by mjuk** [burtonator/polar-bookshelf#285]
It would be nice to be able to go back after clicking on a hyperlink or jumping to another page.
In Firefox PDF viewer this is done with Alt+Left Arrow.

**Comment by Rumata888** [comment]
This would be extremely useful with scientific papers and books.

**Comment by burtonator** [comment]
yeah.. Agreed.. I'm probably going to do something like this at some point via react router so that the user can update the page based on permalinks and then then can be restored.

**Comment by roboticsbrian** [comment]
I've started using Polar for literature review and I would really appreciate this feature!
